Application/EdFi.Ods.Tests/EdFi.Ods.Api/Providers/Aes256SymmetricStringEncryptionProviderTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.Api.Providers;
+using Shouldly;
+
+namespace EdFi.Ods.Tests.EdFi.Ods.Api.Providers;
+
+using System;
+using NUnit.Framework;
+
+public class Aes256SymmetricStringEncryptionProviderTests
+{
+    private Aes256SymmetricStringEncryptionProvider _encryptionProvider;
+    private byte[] _staticKey;
+    [SetUp]
+    public void SetUp()
+    {
+        _encryptionProvider = new Aes256SymmetricStringEncryptionProvider();
+        _staticKey = Convert.FromBase64String("t3OD+5YCKzCdGSaWBAvkQ9+d9R4ZbZnC9YG8L/ar8vA=");
+    }
+
+    [Test]
+    public void When_encrypting_then_decrypting_a_string()
+    {
+        // Arrange
+        var plainText = "Hello World";
+
+        // Act
+        string encryptedString = _encryptionProvider.Encrypt(plainText, _staticKey);
+        bool decryptedSuccessfully = _encryptionProvider.TryDecrypt(encryptedString, out string decryptionResult, _staticKey);
+
+        // Assert
+        decryptedSuccessfully.ShouldBeTrue();

```

Comment:
Consider adding an assertion here that `encryptedString` is not equal to `plainText`.

Reply:
That assertion is added.